.github/workflows/words-to-ignore.txt

```diff
@@ -1,30 +1,41 @@
 CIDs
-Hardt
-Holmgren
+Canonicalization
+Canonicalized
+DIDs
+IPLD
+IPLD-enabled
 JSON
 JWT
 JWT-encoded
+JWTs
 Pre-Draft
 TTL
 UCAN
 UCAN's
 UCANs
+Validators
 Zelenka
 auth
 backend
+canonicalization
+canonicalize
+codec
+conformant
+delimeters
 dholms
+encodings
 expede
+inlined
 interpretable
 multiformat
+namespaced
 requestor's
 responder
+signalled
 signalling
 trustless
+unpadded
 v0
 validator
-
-# wut
```

Comment:
Yeah, sorry 'bout those. Upgrade to version 4.2.2, it fixes a bunch of bugs and works more reliably :sweat_smile: 
("What, you created this project a month ago and it's already on version 4.2.2???" Yeah who would've thought it'd be hard to write a spellchecker LOL)

Reply:
("How hard could it be?")